src/components/Book.jsx

```diff
@@ -0,0 +1,19 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] You did a great job working on this milestone! 🚀 However, please avoid disabling linter rules! Linters are used to maintain a consistent format and help you follow best practices and conventions. Instead, try to fix any possible errors with the rule 🤓 🔧